### ISSUE TITLE
Read more example update/testing

### ISSUE THREAD

**Issue by cehfisher** [cehfisher/a11y-style-guide#164]
One item to note: Describedby (at least with JAWS/NVDA and Firefox) only gets read when you tab to the element. The describedby text isn't read if you arrow through the text for example 3 (via Rachel Olivero Drupal slack).
